script/migrations/mainnet/04-upgrade_bridger.s.sol

```diff
@@ -1,50 +1,47 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.18;
 
-import "../../../src/bridger/Bridger.sol";
+import "@kinto-core/bridger/Bridger.sol";
 
-import "../../../test/helpers/Create2Helper.sol";
-import "../../../test/helpers/ArtifactsReader.sol";
-import "../../../test/helpers/UUPSProxy.sol";
+import {DeployerHelper} from "@kinto-core/libraries/DeployerHelper.sol";
+import {ArtifactsReader} from "@kinto-core-test/helpers/ArtifactsReader.sol";
+import {UUPSProxy} from "@kinto-core-test/helpers/UUPSProxy.sol";
 
 import "forge-std/Script.sol";
 import "forge-std/console.sol";
 import "forge-std/Test.sol";
 
-contract UpgradeBridgerScript is Create2Helper, ArtifactsReader, Test {
-    Bridger bridger;
+contract UpgradeBridgerScript is ArtifactsReader, DeployerHelper, Test {
```

Comment:
shouldnt this file have been named 08-upgrade-bridger?

Reply:
I don't know. We don't have any guidelines on how exactly we deploy. It feels creating a new script for each upgrade is a bit too much overhead, given all these scripts will look exactly the same except the version of the contract. But that is just my take. Happy to follow whatever everyone belive is the best.